ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.14.yaml

```diff
@@ -0,0 +1,181 @@
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate415:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.15"
+  intermediate416:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  latest:
+    release:
+      architecture: multi
+      channel: fast
+      version: "4.14"
+  target:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: aws-ipi-proxy-sts-arm-f28
+  cron: 5 11 5 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE415},${RELEASE_IMAGE_INTERMEDIATE416},${RELEASE_IMAGE_TARGET}"
+        > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE416
+        name: release:intermediate416
+      - env: RELEASE_IMAGE_INTERMEDIATE415
+        name: release:intermediate415
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-chainupgrade
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: azure-ipi-basecap-vset-additionalcap-arm-f28
+  cron: 21 1 12 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+      PERSISTENT_MONITORING: "false"
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE415},${RELEASE_IMAGE_INTERMEDIATE416},${RELEASE_IMAGE_TARGET}"
+        > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE416
+        name: release:intermediate416
+      - env: RELEASE_IMAGE_INTERMEDIATE415
+        name: release:intermediate415
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-chainupgrade
+    workflow: cucushift-installer-rehearse-azure-ipi-capability-baselinecaps-vset-additionalcaps
+- as: gcp-ipi-sdn-arm-f28
```

Comment:
If we want to cover sdn on all platforms, sounds like gcp already have `gcp-upi-private-xpn-sdn-f28` to cover gcp, maybe we can change this one to azure-ipi-sdn ?

Reply:
I see `gcp-upi-private-xpn-sdn-f28` is in 4.13 -> 4.17 upgrade path while `gcp-ipi-sdn-arm-f28` is in 4.14 -> 4.17, and in 4.14 -> 4.17 we already have `azure-ipi-basecap-vset-additionalcap-arm-f28` in multiarch path. So it's not necessary to change `gcp-ipi-sdn-arm-f28` to `azure-ipi-sdn`, right?

Reply:
If that, I wonder what is the test purpose of `gcp-ipi-sdn-arm` ?

Reply:
cover sdn and gcp, then use arm.

Reply:
The same concern as vsphere-ipi-sdn in the above discussion thread, that sounds a duplicate with gcp-upi-private-xpn-sdn-f28 in 4.13 -> 4.17 upgrade path.

Reply:
I think you mean `vsphere-ipi-sdn` in 4.14 -> 4.17 upgrade path? it has already be replaced by `vsphere-ipi-ovn-dualstack-privmaryv6`

Reply:
Here is the new test matrix : https://docs.google.com/spreadsheets/d/12NAZtZas0cHBjO3BlpdM_AuUaoYin7Na68K7SrEHvkA/edit?gid=1470609097#gid=1470609097

Reply:
i think we can reduce the coverage rate of sdn from 4.17 because at that time we have some customers successfully upgraded to 4.16.  but now we have to make sure sdn works and can migrate to ovn.

Reply:
In this PR, we have:
4.12 -> 4.17 azure-ipi-sdn
4.13 -> 4.17 gcp-upi-private-xpn-sdn
4.15 -> 4.17 vsphere-ipi-sdn
4.14 -> 4.17 gcp-ipi-sdn

From https://github.com/openshift/release/pull/53470, actually it just was doing a sdn-to-ovn migration before upgrade, but not some auto-migration during upgrade, and "4.14 -> 4.17 gcp-ipi-sdn" is already duplicated with "4.13 -> 4.17 gcp-upi-private-xpn-sdn" from cluster configuration view. cc @zhaozhanqi do you think we need to cover "sdn-to-ovn migration then upgrade" with this high coverage rate ?

Reply:
> cc @zhaozhanqi do you think we need to cover "sdn-to-ovn migration then upgrade" with this high coverage rate ?
hmm,  I'm working with support team to try to collect the data from customers who will do migrate to OVN.  if there is customer's cluster init version is 4.14 with sdn.  then we may need to cover it.  Because sometimes there maybe a little difference between fresh install cluster 4.14 and (4.14 upgraded from 4.13).. eg .we found mtu configmap is not existing for upgraded cluster.  

if no customer is init install with 4.14.  we can remove it if only for migration testing.  
For now.  I have not got the data yet from customers. so I prefer to keep it if we have

Reply:
hi @zhaozhanqi have you got the date? do we have customers who is using 4.14 + sdn?

Reply:
as @zhaozhanqi said in https://redhat-internal.slack.com/archives/CF8SMALS1/p1720674068714469, we can continue our work and leave sdn jobs along until he get the usage status of 4.14 + sdn.